test/prototype/safetensors/test_safetensors_support.py

```diff
@@ -36,13 +39,26 @@ def load_data(file_path: str, device: str):
 @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
 @unittest.skipIf(not is_sm_at_least_89(), "Need sm89+")
 class TestSafeTensors(TestCase):
-    def test_safetensors(self):
-        config = Float8DynamicActivationFloat8WeightConfig(granularity=PerRow())
+    @parametrize(
+        "config, act_pre_scale",
+        [
+            (Float8DynamicActivationFloat8WeightConfig(granularity=PerRow()), None),
+            (Int4WeightOnlyConfig(), None),
+            (
+                Int4WeightOnlyConfig(),
+                torch.ones((1), dtype=torch.bfloat16),
```

Comment:
probably just construct this in test would be better, just have a True/False flag here